src/test/java/com/quirko/logic/bricks/DotBrickTest.java

```diff
@@ -0,0 +1,31 @@
+package com.quirko.logic.bricks;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DotBrickTest {
+
+    @Test
+    public void testIsShapeDot() throws Exception {
+        Brick brick = new DotBrick();
```

Comment:
Bu neyi test ediyor? DotBrick matrixinde bir yer hariç her yerin sıfır olduğunu yani şeklinin nokta olduğunu test ediyorsun galiba.

Buradaki testleri arttır. DotBrick'in başka şekillere doğru şekilde dönüştürüldüğünün testi yok örneğin. Test kısmına biraz daha kafa yor.